src/main/resources/db/changelog/changes/09-create-cart-items-table.yaml

```diff
@@ -18,11 +18,15 @@ databaseChangeLog:
                   type: bigint
                   constraints:
                     nullable: false
+                    foreignKeyName: fk_cart_items_shopping_carts
+                    references: shopping_carts(id)
               - column:
                   name: book_id
                   type: bigint
                   constraints:
                     nullable: false
+                    foreignKeyName: fk_cart_items_books
+                    references: books(id)
```

Comment:
the same remark as above

Reply:
Okay, I'll take that into account. Now there’s probably no point in adding it separately.